kubernetes/samples/scripts/terraform/oke.delete.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# This script deletes provisioned OKE Kubernetes cluster using terraform (https://www.terraform.io/)
+#
+#
+
+set -o errexit
+set -o pipefail
+
+function prop {
+    grep "${1}" ${oci_property_file}| grep -v "#" | cut -d'=' -f2
+}
+
+function cleanupLB {
+  echo 'Clean up left over LB'
+myvcn_id=`oci network vcn list --compartment-id $compartment_ocid  --display-name=${clusterName}_vcn | jq -r '.data[] | .id'`
```

Comment:
add some indentation for the function body.

Reply:
fixed